lnd_test.go

```diff
@@ -10961,6 +11014,24 @@ func testSendUpdateDisableChannel(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("expected transactions not found in mempool: %v", err)
 	}
 	mineBlocks(t, net, 1)
+
+	// Also do this check for Eve's channel with Carol.
+	ctxt, _ = context.WithTimeout(ctxb, timeout)
+	_, _, err = net.CloseChannel(ctxt, eve, chanPointEveCarol, false)
+	if err != nil {
+		t.Fatalf("unable to close channel: %v", err)
+	}
+
+	waitForChannelUpdate(
+		t, daveUpdates, eve.PubKeyStr, expectedPolicy,
+		chanPointEveCarol,
+	)
+
+	_, err = waitForNTxsInMempool(net.Miner.Node, 1, timeout)
+	if err != nil {
+		t.Fatalf("expected transactions not found in mempool: %v", err)
+	}
+	mineBlocks(t, net, 1)
```

Comment:
Perhaps we should mine more blocks so that the force closed channel is not lingering after the test is over?

Reply:
Good point, have a commit here to generalize this: https://github.com/lightningnetwork/lnd/pull/1621/commits/9c1df170d0bef3acc49cdcf897511c5857d9d18b

Will add it to this test case.